docs/source/en/tutorials/using_peft_for_inference.md

```diff
@@ -12,19 +12,18 @@ specific language governing permissions and limitations under the License.
 
 [[open-in-colab]]
 
-# Inference with PEFT
+# Loading adapters (LoRAs) on diffusers with PEFT for inference
 
-There are many adapters trained in different styles to achieve different effects. You can even combine multiple adapters to create new and unique images. With the 🤗 [PEFT](https://huggingface.co/docs/peft/index) integration in 🤗 Diffusers, it is really easy to load and manage adapters for inference. In this guide, you'll learn how to use different adapters with [Stable Diffusion XL (SDXL)](../api/pipelines/stable_diffusion/stable_diffusion_xl) for inference.
+There are many adapters (commonly known as LoRAs) trained in different styles to achieve different effects. You can even combine multiple adapters to create new and unique images. With the 🤗 [PEFT](https://huggingface.co/docs/peft/index) integration in 🤗 Diffusers, it is really easy to load and manage adapters for inference. In this guide, you'll learn how to use different adapters with [Stable Diffusion XL (SDXL)](../api/pipelines/stable_diffusion/stable_diffusion_xl) for inference.
```

Comment:
Hmm this can indeed be confusing. LoRA is a type of adapter, actually. So, I wonder how to best propagate that.

Reply:
True. But I do think it's fundamental that we include the word LoRA quite high up in such docs, as it's important to optimize for search engine optimization and also for discoverability. People looking to know "how do I load LoRAs with diffusers" need to find this page imo, and they won't be looking for "adapters".